### ISSUE TITLE
MPV problems and dep_ch command not found

### ISSUE THREAD

**Issue by itzCozi** [pystardust/ani-cli#965]
Version: [4.3]
OS: [Windows 10 (using scoop)]
Shell: [Bash]
ScreenShot: Yes and it's super nice
Anime: All 

**Describe the bug**

The function that handles sensing packages is broken I installed MPV and all other packages, 
on line 129 to 136 is a simple function that checks all the package's validity. also in lines 159 to 185 I suspect a bug because when I search for any anime and click on it the script returns an Error like "Episodes not released yet!" 

**Steps To Reproduce**

1. Download [Revo uninstaller](https://www.revouninstaller.com/revo-uninstaller-free-download/)
2. Completely wipe scoop and ani-cli and mpv off your pc
3. Redownload all dependencies 
4. try to run the ani-cli command in cmd(Win10) or git
5. should be hit with a "Program MPV not found please install"
6. remove the dependency check and run again
7. now all we see is "line 549: dep_ch: command not found" and "line 554: dep_ch: command not found"
8. Being the smart cookie i am i tinker in the code and find a way to run this script so we can get error messages
9. after installing vlc with scoop and running ani-cli -v we get the search bar so far so good! then "Episodes not released yet!"

**Tips to fix**

Now I am no one to be telling you kick ass open source devs how to fix your issues but i'll try, after our last attempt(9) i now suspect an api /web scraping problem or search error (Ln 159-185) or maybe since your last patch the file structure of scoop changed and now your looking in the wrong folder, another possibility is when you rout the script to git in the shims directory git has trouble looking for packages because of shells switching from cmd to git. With those and my debugging notes I hope you can get further than me!

[image](https://user-images.githubusercontent.com/93058744/210677042-6874dbdd-3620-467c-8736-657f6b44fade.png)

**Comment by itzCozi** [comment]
Excuse me I am sorry for closing this report I actually want it open.

**Comment by Nannk** [comment]
#959?

**Comment by itzCozi** [comment]
I thought this was a bigger matter due to how it might be caused to faulty packages

**Comment by Derisis13** [comment]
First, why cmd? It's notorius for not being posix compliant so it's not guaranteed to run ani-cli as intended.
Second, git bash has a problem with path. Here's a guide on how to install programs accessible to git bash: https://gist.github.com/evanwill/0207876c3243bbb6863e65ec5dc3f058

**Comment by itzCozi** [comment]
I wasn't considering speed I was using the scoops schims, I just went into scoop parent directory and found the file(s) that where named the same as the command in power shell I changed the file name and ran the new name as a command and it opened ani-cli I was lead to believe that those files hook into the cmd line and run the scripts. I thought this was the best method for windows so I based my approach to the bug on it. Also when i tried to run this on completely different hardware the same error "Episodes not released yet!" is thrown, is anyone using the current version(3.4) without this bug?

**Comment by Nannk** [comment]
> ... Also when i tried to run this on completely different hardware the same error "Episodes not released yet!" is thrown, is anyone using the current version(3.4) without this bug?

regarding "Episodes not released yet": https://github.com/pystardust/ani-cli/issues/948

**Comment by itzCozi** [comment]
I will try this fix when I get home and let you know the results.

**Comment by itzCozi** [comment]
After I read this issue there was not a patch do you know of one?

**Comment by Nannk** [comment]
TL;DR of https://github.com/pystardust/ani-cli/issues/948 is: 
v3 wont work because the site it was scraping is no more. 
ani-cli v4 is in works, is usable, but some features like history are yet to be implemented. Because v4 is not released yet, you would need to clone the [v4 branch](https://github.com/pystardust/ani-cli/tree/v4) to your pc or just copy the [script](https://github.com/pystardust/ani-cli/blob/v4/ani-cli).  I do not know how to do that on windows, so can not tell you exact procedure.

**Comment by itzCozi** [comment]
Ok ill try the latest V4 code in the branch

**Comment by itzCozi** [comment]
I got a "wget: Command not found"

**Comment by itzCozi** [comment]
ill try on linux

**Comment by justchokingaround** [comment]
just install wget

**Comment by itzCozi** [comment]
Ok i installed all the stuff but now when i am "watching" an episode its just black.

**Comment by Derisis13** [comment]
which player are you using?

**Comment by itzCozi** [comment]
mpv

**Comment by Derisis13** [comment]
This issue was going on for a while, and v4 was released in the meantime. The instructions for windows were also updated based on this issue. If he black screen problem still exists please open a new issue for it instead of using this one
